pkg/controllers/networkinfo/networkinfo_utils.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"reflect"
 
-	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	"github.com/zhengxiexie/vsphere-automation-sdk-go/services/nsxt/model"
```

Comment:
is it possible we use replace in go.mod instead of changing this import path everywhere @TaoZou1 @wenqiq

Reply:
we use github.com/vmware/vsphere-automation-sdk-go/ in code files and replace in go.mod. @wenqiq